File: Python/Program/Aksesoris.py
from PetShop import PetShop

class Aksesoris(PetShop):
    def __init__(self, id, nama_produk, harga_produk, stok_produk, jenis, bahan, warna):
        super().__init__(id, nama_produk, harga_produk, stok_produk)
        self.jenis = jenis
        self.bahan = bahan
        self.warna = warna
    
    def get_jenis(self):
        return self.jenis
    
    def get_bahan(self):
        return self.bahan
    
    def get_warna(self):
        return self.warna
    
    def set_jenis(self, jenis):
        self.jenis = jenis
    
    def set_bahan(self, bahan):
        self.bahan = bahan
    
    def set_warna(self, warna):
        self.warna = warna

File: Java/Program/Aksesoris.java
public class Aksesoris extends PetShop {
    protected String jenis;
    protected String bahan;
    protected String warna;

    public Aksesoris(String id, String namaProduk, double hargaProduk, int stokProduk,
                   String jenis, String bahan, String warna) {
        super(id, namaProduk, hargaProduk, stokProduk);
        this.jenis = jenis;
        this.bahan = bahan;
        this.warna = warna;
    }

    public String getJenis() {
        return this.jenis;
    }

    public String getBahan() {
        return this.bahan;
    }

    public String getWarna() {
        return this.warna;
    }

    public void setJenis(String jenis) {
        this.jenis = jenis;
    }

    public void setBahan(String bahan) {
        this.bahan = bahan;
    }

    public void setWarna(String warna) {
        this.warna = warna;
    }
}